tools/container/toolkit/toolkit.go

```diff
@@ -512,6 +598,11 @@ func installToolkitConfig(c *cli.Context, toolkitConfigPath string, nvidiaContai
 		"nvidia-container-runtime-hook.path":                nvidaContainerRuntimeHookPath,
 		"nvidia-container-runtime-hook.skip-mode-detection": opts.ContainerRuntimeHookSkipModeDetection,
 	}
+
+	if len(ctkRuntimes) > 0 {
+		configValues["nvidia-container-runtime.runtimes"] = ctkRuntimes
```

Comment:
Should we allow a user to specify these directly?

Reply:
I have made a proposal for constructing this option -- if not set -- in `run.go` instead of here.